superset-frontend/src/components/ListViewCard/ImageLoader.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+
+interface ImageLoaderProps
+  extends React.DetailedHTMLProps<
+    React.ImgHTMLAttributes<HTMLImageElement>,
+    HTMLImageElement
+  > {
+  fallback: string;
+  src: string;
+  isLoading: boolean;
+}
+
+export default function ImageLoader({
+  src,
+  fallback,
+  alt,
+  isLoading,
+  ...rest
+}: ImageLoaderProps) {
+  const [imgSrc, setImgSrc] = React.useState<string>(fallback);
+
+  React.useEffect(() => {
```

Comment:
This seems fine, but often people just use the named import of `useEffect` - I'm not sure if we have a pattern we want to follow/establish